src/library/simplified/extension/amenities.js

```diff
@@ -0,0 +1,16 @@
+var offeredAmenities = '';
+if(b.entity.checkout.hotel != undefined){
+    offeredAmenities = b.entity.checkout.hotel.offeredAmenities;
+}
+else if(b.entity.checkout.hotels != undefined){
+    offeredAmenities = b.entity.checkout.hotels[0].offeredAmenities;
+}
+if(offeredAmenities != ''){
+    var offeredAmenitiesNumber = offeredAmenities.length;
```

Comment:
give proper name for offeredAmenitiesNumber and use proper variable name insteaf of i